File: src/test/java/br/com/newObjects/model/entities/classes/NovaAvaliacao.java
package br.com.newObjects.model.entities.classes;

import br.com.model.entities.classes.Avaliacao;
import br.com.model.entities.classes.Pedido;

public class NovaAvaliacao {

    public static void novaAvaliacao(Pedido pedido) {
        new Avaliacao(5, pedido, "teste de comentário");
    }
}

File: src/test/java/br/com/newObjects/model/entities/classes/NovaAvaliacao.kt
package br.com.newObjects.model.entities.classes

import br.com.model.entities.classes.Avaliacao
import br.com.model.entities.classes.Pedido

object NovaAvaliacao {
    fun novaAvaliacao(pedido: Pedido) {
        Avaliacao(5, pedido, "teste de comentário")
    }
}
